src/geophires_x/Economics.py

```diff
@@ -556,7 +556,7 @@ def CalculateLCOELCOHLCOC(self, model: Model) -> tuple:
             NPVgrt = self.GTR.value / (1 - self.GTR.value) * (NPVcap + NPVoandm + NPVfc + NPVit - NPVitc)
             LCOH = (NPVcap + NPVoandm + NPVfc + NPVit + NPVgrt - NPVitc) / np.sum(
                 model.surfaceplant.HeatkWhProduced.value * inflationvector * discountvector) * 1E8
-            LCOH = self.LCOH.value * 2.931  # $/MMBTU
+            LCOH = LCOH * 2.931  # $/MMBTU
```

Comment:
This change doesn't seem possible without impacting example outputs, but no example output changes are in the PR?

Reply:
This was a one line bug I found some weeks ago and committed but forgot to do a PR, so it got folded into this PR. This bug showed itself in only a couple of cases, but the fix is correct. There may be some output changes, but maybe not.